change/@azure-msal-angular-bd9bc5d3-a8d8-4d07-83b4-246e0a84455f.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "updating package-lock",
```

Comment:
Can you please remove the package lock changes?

Reply:
Removed.